cds/config.py

```diff
@@ -93,6 +93,10 @@ def _(x):
         'task': 'invenio_accounts.tasks.clean_session_table',
         'schedule': timedelta(days=1),
     },
+    'tasks_status': {
+        'task': 'cds.modules.deposit.tasks.preserve_celery_states_on_db',
+        'schedule': timedelta(days=7),
```

Comment:
we could be searching directly on posters for deposits in draft state, they should be only a few.